### PR TITLE
Splash screen support

### PR DESCRIPTION
**Pull request by Cightline**
I figured out how to set the splash screen and have issued a pull request. I also deleted the __pycache__ directory.

### CODE REVIEW

**Comment by dinofizz** [comment]
Hey! Thanks for taking the time to look into this! I will take a look myself at these changes this weekend.

File: lcdbackpack/lcdbackpack.py [hunk 1]
```diff
@@ -234,6 +233,17 @@ def write(self, string):
 
         self._ser.write(str.encode(string))
 
+    def set_splash_screen(self, string, lcd_chars):
+        """
+        Sets the LCD splash screen. 
+        :param string: the text to be displayed at LCD start up
+        :param lcd_chars: the total characters of the LCD display
+        """
+        self._write_command([0x40])
```
Comment (dinofizz) [inline lcdbackpack/lcdbackpack.py]: I would rather use a "constant" for this command as those defined from line 32 in this file.

File: lcdbackpack/lcdbackpack.py [hunk 2]
```diff
@@ -234,6 +233,17 @@ def write(self, string):
...
+        self._ser.write('{{0: <{}}}'.format(lcd_chars).format(string).encode())
```
Comment (dinofizz) [inline lcdbackpack/lcdbackpack.py]: I'll need to unpack exactly what this is doing! My python-fu is not so strong.

File: lcdbackpack/lcdbackpack.py [hunk 3]
```diff
@@ -234,6 +233,17 @@ def write(self, string):
...
+        self._ser.write(bytes(0x00))
```
Comment (dinofizz) [inline lcdbackpack/lcdbackpack.py]: What is the significance of sending this?

**Review by dinofizz — CHANGES_REQUESTED** [review]
Thanks for the PR! I will play with your changes to understand them a bit better myself this weekend. Please review my feedback and let me know what you think?

File: lcdbackpack/lcdbackpack.py (hunk 2 above)
Comment (Cightline) [reply lcdbackpack/lcdbackpack.py]: This is mostly taken from: https://stackoverflow.com/a/5676884

**Comment by Cightline** [comment]
Ok, I think I fixed everything you wanted. And you're welcome for the PR, this is definitely one of the better LCD/Python libraries out there.

**Review by dinofizz — APPROVED** [review]

**Pull request merged**